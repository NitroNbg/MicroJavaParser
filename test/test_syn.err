
Sintaksna greÅ¡ka u liniji 2

Sintaksna greÅ¡ka u liniji 10

Sintaksna greÅ¡ka u liniji 16
UoÄ?ena je greÅ¡ka prilikom sintaksne analize, prevoÄ‘enje je prekinuto
