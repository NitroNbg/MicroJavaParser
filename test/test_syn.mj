program test_syn
	int prva s, prava;
	char druga;
	{
		void pravilna(int x) int zbir; {
			zbir = x + prva;
			return zbir;
		}
		int nepravilna(int x) int zbir; {
			zbir = x + prva +;
			return zbir;
		}
		void main() {
			prava = 0;
			prava = nepravilna(2);
			prava = pravilna(3) s;
			print(prava);
		}
	}