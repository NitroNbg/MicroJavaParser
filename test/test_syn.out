PrevoÄ‘enje izvorne datoteke: C:\MJCompiler\test\test_syn.mj
